Releases/Manifests/5.15.0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "FirebaseAuth":"5.2.0",
+  "FirebaseCore":"5.1.10",
+  "FirebaseFirestore":"0.16.1",
+  "FirebaseFunctions":"2.1.2",
+  "FirebasePerformance":"2.2.2",
+  "Google-Mobile-Ads-SDK":"7.37.0"
```

Comment:
Probably harmless to include Google-Mobile-Ads-SDK, but this should only be the open source pods.

Reply:
This explains why I had some release tooling scripts output commands for this pod. I'll remove it.